Polish/przygoda/jedzenie/wiedzmin/placki/igni/igni.md

```diff
@@ -0,0 +1,7 @@
+Rzucasz zaklecie IGNI - z twoich rak strzelaja plomienie ognia, ktore kierujesz na nerda, przy okazji podpalajac meble stojace wzdluz sciany. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.